lstchain/mc/sensitivity.py

```diff
@@ -179,10 +177,17 @@ def calculate_sensitivity_lima(n_excesses, n_background, alpha, n_bins_energy, n
 
     """
 
+    if any(len(a) != n_bins_energy for a in (n_on_events, n_background, alpha)):
```

Comment:
Hi @lfoffano, Travis is still complaining about this line. If I'm not wrong, `calculate_sensitivity_lima` is a function that is expecting `int/float` because it calculates the LiMa significance for a single energy bin, so this `if` statement will fail as reported in:
https://travis-ci.org/github/cta-observatory/cta-lstchain/jobs/727298230